### ISSUE TITLE
in the main readme, explain the general folder structure

### ISSUE THREAD

**Issue by aschweigert** [INN/Largo-Sample-Child-Theme#2]
including what goes where and why

**Comment by rnagle** [comment]
I filled this out a bit: https://github.com/INN/Largo-Sample-Child-Theme#theme-directory-structure

This could use an edit and perhaps some suggestions on other details to add.

I tried my best not to cite reasons like "because" for the directory structure. ^_^

**Comment by 0aveRyan** [comment]
This has been updated, will continue to add over the Midwest Energy project.
https://github.com/INN/Largo-Sample-Child-Theme/tree/master/docs

**Comment by aschweigert** [comment]
This is looking pretty good to me. A couple things to maybe add/change:
- we've had a couple of site straight up clone this repo and try to start from this instead of from an empty theme and adding only what they really need for their child theme. Is there some way we can discourage that?
- the diagram here is pretty good but a little confusing because the child theme directory is at the same level as wp-content/themes (since they're aligned vertically). Can we maybe try to make this hierarchy clearer (or just go with a simpler directory tree diagram?)

**Comment by 0aveRyan** [comment]
@aschweigert if this is more to serve as a sample, should we have this repo -- a fully fleshed out sample child theme -- and a Largo-Starter-Child-Theme that is a clone-able skeleton for others and us to use?
- [ ] Cleanup diagram
- [ ] Reframe docs to indicate child theme is a sample, not clonable starter.

**Comment by aschweigert** [comment]
maybe? it would be nice to have an additional skeleton repo we can clone when we bootstrap a new site but it really wouldn't have much in it (probably just the less/css directories and child-style.less/css, gruntfile, gitignore...but that's about it. would that be useful or not?

**Comment by 0aveRyan** [comment]
I think for some people with rudimentary CSS and HTML experience, the gruntfile, the gitignore, the functions.php and some base functions, setting up the directory structure -- that stuff is hard. They just want to be able to change the colors of things, maybe a few fonts.

If we did setup our own skeleton, we could have a `master` branch for public consumption and an `inn` branch for use internally. Nothing crazy, pretty sparse like you said.

**Comment by aschweigert** [comment]
right, I'd say that the master branch of that skeleton would likely ONLY have the style.css with the appropriate header block to use it as a child theme and then a readme explaining how this other stuff would work should they need it.
